pkg/dss/scd/server.go

```diff
@@ -22,6 +22,12 @@ type Server struct {
 	Timeout time.Duration
 }
 
+// AuthScopes returns a map of endpoint to required Oauth scope.
+func (a *Server) AuthScopes() map[auth.Operation][]auth.Scope {
+	// TODO: Fill in details here.
```

Comment:
Since we're deleting the existing scope declarations from rid/server.go, can we add them here in this PR so they're not lost?  At a minimum, I'd move the existing declarations over here and comment them all out so this TODO can be more easily fulfilled.

Reply:
Done.